backend/src/yeomanui.ts

```diff
@@ -221,7 +187,8 @@ export class YeomanUI {
 				logger: this.logger.getChildLogger({ label: generatorName }),
 				vscode: this.getVscode(), // TODO: remove this temporary workaround once a better solution is found,
 				data: this.uiOptions.data,
-				swaTracker: SWA.getSWATracker()
+				swaTracker: SWA.getSWATracker(),
```

Comment:
maybe not in this change, but later we should consider move vscode & swaTracker in appWizard.

Reply:
vscode should be removed at all (in the future)
not sure about swaTracker